ssn/chapters/ModelFOI.html

```diff
@@ -1,23 +1,37 @@
-<h3>Ultimate feature of interest</h3>
+<h3>Proximate and Ultimate feature of interest</h3>
 <p>
     The object of the
-    <a href="#SOSAhasFeatureOfInterest">hasFeatureOfInterest</a> property of an act
-    of observation, sampling, or actuation is the <i>immediate</i> or <i>proximate</i>
-    <a href="https://www.w3.org/TR/vocab-ssn/#SOSAFeatureOfInterest">FeatureOfInterest</a>.
-    In some cases this is not the the ultimate thing that the act of observation, sampling, or actuation is concerned
-    with, but is an intermediate thing, often a <i>sample</i> of the ultimate feature of interest, or perhaps a sample
-    of a sample, etc.
-    There will often be a specifiable relationship between the proximate and ultimate feature of interest, such as a
+    <a href="#SOSAhasFeatureOfInterest">hasFeatureOfInterest</a> property of an <a href="#SOSAExecution">Execution</a>
+    (i.e Actuation, Observation,
+    or Sampling) is the <i>immediate</i> or <i>proximate</i> <a
+        href="https://www.w3.org/TR/vocab-ssn/#SOSAFeatureOfInterest">FeatureOfInterest</a>.
+    For example, <a href="#open-window"></a> shows a description of a simple actuation to open a window.
+</p>
+<figure id="open-window">
+    <img alt="Observation feature-of-interest patterns 1" src="./images/open-window.png" style="width:75%" />
+    <figcaption>
+        Simple Actuation to open a window.
+        <br>
+        <em><a href="#class-diagram">Explanation of the notation used in class diagrams</a></em>.
+    </figcaption>
+</figure>
+<pre class="example" data-include="./rdf/examples/open-window.ttl" data-include-format="text"></pre>
+
+<p>
+    In some cases the feature-of-interest of the execution is not the the ultimate thing that the act of actuation,
+    observation, or sampling is concerned with, but is an intermediate thing.
+    This might be a <i>sample</i> of the ultimate feature of interest, or perhaps a sample of a sample, etc.
```

Comment:
```suggestion
    In some cases, the feature-of-interest of the execution is not the the ultimate thing that the act of actuation,
    observation, or sampling is concerned with, but is an intermediate thing.
    This might be a <i>sample</i> of the ultimate feature of interest, or a sample of a sample, etc.
```